### PR TITLE
Fix button alignment in homepage in mobile view

### PR DESCRIPTION
**Pull request by rishipurwar1**
Fixes # 1098

#### Describe the changes you have made in this PR -
Align " for teacher and for contributor button" in center in mobile view

### Screenshots of the changes (If any) -
Before fix

bug2

After fix

bug1099

### CODE REVIEW

**Comment by rishipurwar1** [comment]
please review my pr @tachyons @nitin10s @aayush-05

**Comment by rishipurwar1** [comment]
Ok, I'll update it

**Comment by rishipurwar1** [comment]
@nitin10s  Added margin in between the buttons and in the next section.

**Comment by rishipurwar1** [comment]
@nitin10s Add a margin between the buttons and in the next section

File: app/assets/stylesheets/home.scss [hunk 1]
```diff
@@ -89,3 +89,18 @@
   max-height: 700px;
   max-width: 100%;
 }
+
+@media (max-width:765px) {
+  .row1 {
```
Comment (tachyons) [inline app/assets/stylesheets/home.scss]: Please chose better name instead `.row1`

Comment (rishipurwar1) [reply app/assets/stylesheets/home.scss]: I changed to button-homepage. 
Please check it @tachyons

**Pull request merged**